api/swagger.yaml

```diff
@@ -518,6 +539,19 @@ components:
           items:
             $ref: '#/components/schemas/power_status'
 
+    power_status_get:
+      type: object
+      description: |
+        Filters to limit which nodes have their power status returned.
+      properties:
+        xname:
```

Comment:
can this be xnames instead of xname?
I think capmc also used the plural xnames

Reply:
I just did it to match what the `GET` request already uses. I agree that `xnames` makes more sense, but it would seem worse to have the names differ between the `GET` and the `POST`. And changing it for both would mean this is a breaking change.

Reply:
Talked to Shane on Slack. He has confirmed that he can live with the non-ideal name, but he made it clear that I owe him big time, which I acknowledge.